src/test/java/edu/berkeley/ground/util/TestEnv.java

```diff
@@ -0,0 +1,16 @@
+package edu.berkeley.ground.util;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+public class TestEnv {
```

Comment:
It's probably good to make this `final` too, since static classes shouldn't be instantiated.